### ISSUE TITLE
Add suggestions to be responsible and self-isolate

### ISSUE THREAD

**Issue by catherio** [microCOVID/microCOVID#76]
"I think advising people that if they engage in a high risk activity, they need to be responsible and self-isolate for some period also has value - in other words, they might want to take the personal risk but have a responsibility not to endanger others with their choice"

**Comment by Radvendii** [comment]
I agree. I hear people discussing this a lot as just a personal risk management issue and not one of communal safety.
